### ISSUE TITLE
Remove Batch Norm?

### ISSUE THREAD

**Issue by verbiiyo** [werner-duvaud/muzero-general#203]
### Search before asking

- [X] I have searched the MuZero [issues](https://github.com/werner-duvaud/muzero-general/issues) and found no similar feature requests.

### Description

I noticed batch norm is being used in some of the model architectures used. I also read that [batch norm shouldn't be used in RL settings](https://rdednl.github.io/blog/batch-norm-rl/). Could be promising to boost performance?
